src/NextFederationPlugin.js

```diff
@@ -21,10 +21,12 @@ import {
   getOutputPath,
   externalizedShares,
   removePlugins,
-  parseRemotes
+  parseRemotes,
 } from './internal';
 
 import ChildFriendlyModuleFederationPlugin from './ModuleFederationPlugin';
+import StreamingTargetPlugin from '../node-plugin/streaming';
```

Comment:
we want to keep these are inline requires so users don't have to install webpack unless its using ssr.

Reply:
csr also needs webpack at this point
for example, using templating in image loader patcher:
https://github.com/module-federation/nextjs-mf/blob/c90a4c7d911a1de18314e917427fe0e1a6b74ace/src/loaders/fixImageLoader.js#L1

Reply:
For CSR, we are now including `webpack/lib/template` into the bundle.

https://github.com/module-federation/nextjs-mf/blob/main/packages/nextjs-mf/utils/Template.ts

So far, CSR doesn't need webpack to be installed.